src/Css/Media.elm

```diff
@@ -0,0 +1,1168 @@
+module Css.Media
+    exposing
+        ( MediaQueryComponent
+        , MediaQuery
+        , media
+        , mediaQuery
+        , withMedia
+        , withMediaQuery
+        , only
+        , not
+        , or
+        , all
+        , screen
+        , print
+        , speech
+        , braille
+        , embossed
+        , handheld
+        , projection
+        , tty
```

Comment:
@rtfeldman Sorry for like the 3 month later followup here. Given their deprecation, do you think I should I remove them entirely or do you like you did for some other deprecated CSS things (but not all deprecated things) and have them be available, but unable to ever compile?